src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/tab.scss

```diff
@@ -0,0 +1,63 @@
+@import '../../containers/Application/colors.scss';
+
+$tabTextColor: $black;
+$tabUnderlineBackgroundColor: $black;
+
+.tab {
+    position: relative;
+
+    &:first-child {
```

Comment:
Why has this been solved so complicated? Why doesn't the first tab just have the same paddings etc. as the other tabs, and the tabs class has therefore less padding?